modules/schematics/src/effect/files/__name@dasherize@if-flat__/__name@dasherize__.effects.ts

```diff
@@ -1,12 +1,30 @@
 import { Injectable } from '@angular/core';
 import { Actions, Effect<% if (feature) { %>, ofType<% } %> } from '@ngrx/effects';
-<% if (feature) { %>import { <%= classify(name) %>ActionTypes } from '<%= featurePath(group, flat, "actions", dasherize(name)) %><%= dasherize(name) %>.actions';<% } %>
+<% if (feature && api) { %>import { catchError, map, concatMap } from 'rxjs/operators';<% } %>
+<% if (feature && api) { %>import { EMPTY, of } from 'rxjs';<% } %>
+<% if (feature && api) { %>import { Load<%= classify(name) %>sFailure, Load<%= classify(name) %>sSuccess, <%= classify(name) %>ActionTypes, <%= classify(name) %>Actions } from '<%= featurePath(group, flat, "actions", dasherize(name)) %><%= dasherize(name) %>.actions';<% } %>
+<% if (feature && !api) { %>import { <%= classify(name) %>ActionTypes } from '<%= featurePath(group, flat, "actions", dasherize(name)) %><%= dasherize(name) %>.actions';<% } %>
 
 @Injectable()
 export class <%= classify(name) %>Effects {
-<% if (feature) { %>
+<% if (feature && api) { %>
+  @Effect()
+  load<%= classify(name) %>s$ = this.actions$.pipe(
+    ofType(<%= classify(name) %>ActionTypes.Load<%= classify(name) %>s),
+    concatMap(() =>
+      EMPTY.pipe(
```

Comment:
I feel like we need some comment here to tell the user to replace the empty observable?

```suggestion
      /** An EMPTY observable only emits completion. Replace with your own observable API request */
      EMPTY.pipe(
```

Reply:
That makes sense to me.